py-polars/tests/unit/namespaces/test_plot.py

```diff
@@ -39,8 +38,3 @@ def test_series_hist() -> None:
 
 def test_empty_dataframe() -> None:
     pl.DataFrame({"a": [], "b": []}).plot.scatter(x="a", y="b")
-
-
-def test_unsupported_dtype() -> None:
-    with pytest.raises(PolarsPanicError):
-        pl.DataFrame({"a": [{1, 2}], "b": [4]}).plot.scatter(x="a", y="b")
```

Comment:
Why was this test removed?

Reply:
AFAICT it was originally panicking due to using a column with `Object` dtype (column `"a"` is a set). I get inconsistent errors now: TypeError in Jupyter, and no error when run in a test.

Reply:
Ahh it's a set - I thought it was a dict and thus a struct column. Makes sense then :+1: